eng/native/configureplatform.cmake

```diff
@@ -392,10 +392,10 @@ if(CLR_CMAKE_TARGET_OS STREQUAL SunOS)
     set(CLR_CMAKE_TARGET_SUNOS 1)
 endif(CLR_CMAKE_TARGET_OS STREQUAL SunOS)
 
-if(CLR_CMAKE_TARGET_OS STREQUAL Emscripten)
+if(CLR_CMAKE_HOST_OS STREQUAL Emscripten)
```

Comment:
It was just because sometimes the taget is `Browser` and sometimes `browser`.  I did have some lower case logic in here at one point, i could put that back instead of this?

Reply:
Hmm, so would the following would be equivalent?
```
if(CLR_CMAKE_TARGET_OS STREQUAL browser OR CLR_CMAKE_TARGET_OS STREQUAL Browser)
    set(CLR_CMAKE_TARGET_UNIX 1)
    set(CLR_CMAKE_TARGET_BROWSER 1)
endif(CLR_CMAKE_TARGET_OS STREQUAL browser OR CLR_CMAKE_TARGET_OS STREQUAL Browser)
```
If so it would be better to do this (or normalize somewhere above).

Evidently the convention here is to require exact casing of target OSes when passing them to CMake, so "the best" solution would be to make sure it is always `Browser` but I do not know how costly that would be.

Reply:
One place that's immediately apparent is `build-runtime.cmd` here:

https://github.com/dotnet/runtimelab/blob/10a88e02a5b239693af0f63a060659818d03ec35/src/coreclr/build-runtime.cmd#L183-L186

Reply:
Looks like that was the only one that needs changing